tree/ntuple/src/RFieldMeta.cxx

```diff
@@ -1151,6 +1166,22 @@ ROOT::RTupleField::RTupleField(std::string_view fieldName, std::vector<std::uniq
    }
 }
 
+void ROOT::RTupleField::ReconcileOnDiskField(const RNTupleDescriptor &desc)
+{
+   static const std::vector<std::string> prefixes = {"std::pair<", "std::tuple<"};
```

Comment:
We should probably change `EnsureMatchingTypePrefix` to accept a `span` so we can avoid creating `vector`s for things that should really be static arrays